examples/from_googlesheets.py

```diff
@@ -0,0 +1,26 @@
+import pandas as pd
+from data.sample_dataframe import dataframe
+
+from pandasai import PandasAI
+from pandasai.helpers.datasource import sheets_input
+from pandasai.llm.openai import OpenAI
+
+# instructions: https://docs.google.com/document/d/1uAmXI3e8Wm26lZQb0bYIJaXRdZi5WLJKNJHT59QMGEc/edit?usp=sharing
+df = sheets_input(
+    "1IRGJFUpCR0-9tPDk2MgPjQCtNft_4a8B1fHgYK7_LM8",
+    "second",
+    "../service_account_file.json",
+)
+# specify type of columns, google sheets is not strongly typed
+df["gdp"] = pd.to_numeric(df["gdp"])
+df["happiness_index"] = pd.to_numeric(df["happiness_index"])
```

Comment:
This should be automatically done by default as we run `from_google_sheets `, what do you think? Otherwise having to manually make this transformation all the times could be very repetitive.